acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/GitLabPages.scala

```diff
@@ -34,6 +39,9 @@ class GitLabPages(
       s"/gitlab/${userCredentials.userNamespace}/${projectDetails.title.toPathSegment}"
     )
 
+    // val branchesPath: String =
```

Comment:
Do we still need the commented out code?

Reply:
Removed.